### PR TITLE
Change default tiling 512

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

File: services/src/server.rs [hunk 1]
```diff
@@ -34,15 +34,26 @@ pub async fn start_server(static_files_dir: Option<PathBuf>) -> Result<()> {
...
+    .await;
+
+    let chunk_byte_size = config::get_config_element::<config::QueryContext>()?
+        .chunk_byte_size
+        .into();
+
+    let tiling_spec = config::get_config_element::<config::TilingSpecification>()?.into();
+
+    ctx.set_chunk_byte_size(chunk_byte_size);
+    ctx.set_tiling_spec(tiling_spec);
```
[the remark this answers is not in the document]
Comment (jdroenner) [reply services/src/server.rs]: 8672114

File: services/src/server.rs [hunk 2]
```diff
@@ -34,15 +34,26 @@ pub async fn start_server(static_files_dir: Option<PathBuf>) -> Result<()> {
...
+    )
+    .await;
+
+    let chunk_byte_size = config::get_config_element::<config::QueryContext>()?
+        .chunk_byte_size
+        .into();
+
+    let tiling_spec = config::get_config_element::<config::TilingSpecification>()?.into();
+
+    ctx.set_chunk_byte_size(chunk_byte_size);
```
[the remark this answers is not in the document]
Comment (jdroenner) [reply services/src/server.rs]: 8672114

File: services/src/handlers/workflows.rs [hunk 3]
```diff
@@ -976,7 +980,9 @@ mod tests {
         .boxed();
 
         let session = ctx.default_session_ref().await.clone();
-        let exe_ctx = ctx.execution_context(session).unwrap();
+        let mut exe_ctx = ctx.execution_context(session).unwrap();
+        // override the pixel size since this test was designed for 600 x 600 pixel tiles
+        exe_ctx.tiling_specification.tile_size_in_pixels = GridShape::new([600, 600]);
```
[the remark this answers is not in the document]
Comment (jdroenner) [reply services/src/handlers/workflows.rs]: 8672114

File: services/src/handlers/workflows.rs [hunk 4]
```diff
@@ -976,7 +980,9 @@ mod tests {
         .boxed();
 
         let session = ctx.default_session_ref().await.clone();
-        let exe_ctx = ctx.execution_context(session).unwrap();
+        let mut exe_ctx = ctx.execution_context(session).unwrap();
+        // override the pixel size since this test was designed for 600 x 600 pixel tiles
```
[the remark this answers is not in the document]
Comment (jdroenner) [reply services/src/handlers/workflows.rs]: 8672114

File: services/src/handlers/wfs.rs [hunk 5]
```diff
@@ -1060,7 +1061,12 @@ x;y
     async fn raster_vector_join() {
         dir_up();
 
-        let ctx = InMemoryContext::default();
+        let mut ctx = InMemoryContext::default();
```
[the remark this answers is not in the document]
Comment (jdroenner) [reply services/src/handlers/wfs.rs]: 8672114

File: services/src/handlers/wms.rs [hunk 6]
```diff
@@ -533,7 +539,12 @@ mod tests {
     ///Actix uses serde_urlencoded inside web::Query which does not support this
     #[tokio::test]
     async fn get_map_uppercase() {
-        let ctx = InMemoryContext::default();
+        let mut ctx = InMemoryContext::default();
```
[the remark this answers is not in the document]
Comment (jdroenner) [reply services/src/handlers/wms.rs]: 8672114

File: services/src/handlers/datasets.rs [hunk 7]
```diff
@@ -1011,19 +1009,27 @@ mod tests {
     async fn create_dataset() -> Result<()> {
         let mut test_data = TestDataUploads::default(); // remember created folder and remove them on drop
 
-        let ctx = InMemoryContext::default();
+        let mut ctx = InMemoryContext::default();
+        // override the pixel size since this test was designed for 600 x 600 pixel tiles
```
[the remark this answers is not in the document]
Comment (jdroenner) [reply services/src/handlers/datasets.rs]: 8672114

File: services/src/contexts/mod.rs [hunk 8]
```diff
@@ -84,18 +81,25 @@ where
     dataset_db: Db<D>,
     thread_pool: ThreadPoolContext,
     session: S,
+    pub tiling_specification: TilingSpecification,
```
[the remark this answers is not in the document]
Comment (jdroenner) [reply services/src/contexts/mod.rs]: changed that in 8672114

File: operators/src/processing/point_in_polygon.rs [hunk 9]
```diff
@@ -241,8 +241,11 @@ impl VectorQueryProcessor for PointInPolygonFilterProcessor {
                 });
 
         Ok(
-            FeatureCollectionChunkMerger::new(filtered_stream.fuse(), ctx.chunk_byte_size())
-                .boxed(),
+            FeatureCollectionChunkMerger::new(
+                filtered_stream.fuse(),
+                ctx.chunk_byte_size().inner(),
```
[the remark this answers is not in the document]
Comment (jdroenner) [reply operators/src/processing/point_in_polygon.rs]: a571e90

**Review by michaelmattig — APPROVED** [review]

**Comment by michaelmattig** [comment]
bors merge

**Pull request merged**